src/Directory.Build.props

```diff
@@ -2,26 +2,15 @@
 
   <PropertyGroup>
     <Authors>Jake Ginnivan</Authors>
-    <PackageProjectUrl>https://github.com/PublicApiGenerator/PublicApiGenerator</PackageProjectUrl>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <Copyright>Copyright Jake Ginnivan 2010-$([System.DateTime]::UtcNow.ToString(yyyy)). All rights reserved.</Copyright>
     <PackageTags>semanticversioning versioning api</PackageTags>
-    <PackageOutputPath>..\..\nugets</PackageOutputPath>
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
-  </PropertyGroup>
-
-  <PropertyGroup>
     <MinVerMinimumMajorMinor>10.0</MinVerMinimumMajorMinor>
-    <NoWarn>NU5105</NoWarn>
+    <NoWarn>$(NoWarn);NU5105</NoWarn>
+    <IsPackable>true</IsPackable>
```

Comment:
Making projects "packable" by default seems a little dangerous. Say that in the NuGet publishing step, we simply scan for all `*.nupkg` and publish them, and say that for some reason we add a second unit test project in the future but we forget to explicitly set its `<IsPackable>` property to `False`; we'd then publish the unit tests accidentally, wouldn't we?

I believe it would be safer to explicitly opt-in to `<IsPackable>`, rather than having to opt-out.

Reply:
In fact, I spent a lot of time to comprehend what strategy for this property (in conjunction with others) is the most correct and universal for any .NET repo. There are several variations that one way or another lead to the desired result. For such a small project as this I can apply another option that you are talking about.

Reply:
done